repository/repository-core/pom.xml

```diff
@@ -246,6 +251,20 @@
             <version>2.0.2.RELEASE</version>
             <scope>test</scope>
         </dependency>
+        
+        <!-- Elastic search indexing service (start) -->
+        <dependency>
```

Comment:
Hi @erlemantos 
What is this dependency for ?

Reply:
A dependency to the elastic core and dependency to the high level client of Elastic cluster

Reply:
I see.